mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundOperations.tsx

```diff
@@ -34,41 +37,80 @@ export function PlaygroundOperations (): JSX.Element {
     return await account.getAddress()
   }
 
+  const dataLists = [
+    {
+      title: 'Fetch balances',
+      onPress: async (): Promise<void> => {
+        const address = await getActiveAddress()
+        dispatch(fetchTokens({
+          client,
+          address
+        }))
+      },
+      rhsChildren: (): JSX.Element => {
+        return (
+          <ThemedIcon
+            light={tailwind('text-mono-light-v2-700')}
+            dark={tailwind('text-mono-dark-v2-700')}
+            iconType='Feather'
+            name='arrow-down-circle'
+            size={18}
+          />
+        )
+      },
+      testID: 'playground_wallet_fetch_balances'
+    },
+    {
+      title: 'Generate block',
+      onPress: async (): Promise<void> => {
+        await rpc.call('generatetoaddress', [10, 'mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy'], 'number')
+      },
+      rhsChildren: (): JSX.Element => {
+        return (
+          <ThemedIcon
```

Comment:
It should be + icon here based on design

Reply:
Same with the other top up tokens 
![image](https://user-images.githubusercontent.com/13292662/185825288-0a56b936-ea0a-46a5-a2e7-602eac762e8e.png)